pkg/autodiscovery/listeners/docker.go

```diff
@@ -137,11 +141,18 @@ func (l *DockerListener) init() {
 		}
 		var svc Service
 
+		checkNames, err := getCheckNamesFromLabels(co.Labels)
+		if err != nil {
+			log.Errorf("Error getting check names from docker labels: %v", err)
```

Comment:
maybe you can add the container name in the error to ease investigation

Reply:
makes sense 👍